website/blog/2025-05-22-release-1.19.md

```diff
@@ -0,0 +1,396 @@
+---
+title: Podman Desktop 1.19 Release
+description: Podman Desktop 1.19 has been released!
+slug: podman-desktop-release-1.19
+authors: [axel7083]
+tags: [podman-desktop, release, kubernetes, openshift]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.19.png
+---
+
+import ThemedImage from '@theme/ThemedImage';
+
+Podman Desktop 1.19 Release! 🎉
+
+![podman-desktop-hero-1.19](/img/blog/podman-desktop-release-1.19/banner.png)
+
+Podman Desktop 1.19 is now available. [Click here to download it](/downloads)!
+
+This release among many extension update brings exciting new features and improvements:
+
+- **Podman v5.5**: The Podman executable shipped with Podman Desktop has been updated to its latest version.
+- **Bootc Extension 1.9**: Test and experiment with Bootable Container directly from the extension.
+- **AI Lab 1.7.2**: New inference runtime, llama-stack implementation and MCP.
+- **Minc extension**: Podman Desktop extension to deploy MicroShift as container.
+- **RHEL lightspeed extension**: Extension to get assistance using RHEL Lightspeed.
+- **RHEL VMs extension**: The RHEL VMs extension helps the user run RHEL Virtual machines.
+
+## Release details 🔍
+
+### Podman v5.5
+
+Podman Desktop aims to work offline, so it ship alongside its binary, the podman executable.
+Including in this release, the new version of Podman 5.5 will be included, providing new features, bug fixes and a better stability while using the tool.
+
+You can found the full changelog of Podman 5.5 on their release page [containers/podman/releases#v5.5.0](https://github.com/containers/podman/releases/tag/v5.5.0)
+
+:::info
+
+While the podman binary is updated in this update, the existing Podman Machines will still stay in the version where have been created.
+
+To update your podman machine(s) you need to remove them and init a new one.
+
+:::
+
+### Bootc Extension 🦭
+
+The Podman Desktop Boot container extension (Aka Bootc) has seen recently some big improvement, one of them is the usage of [macadam](https://github.com/crc-org/macadam).
+
+> Macadam is a cross-platform command-line tool to create and manage virtual machines. It runs on Windows, macOS and linux, and uses each OS native virtualization stacks, that is WSL2 on Windows, Apple’s Virtualization Framework on macOS and QEMU on linux. macadam is reusing podman-machine code.
+
+As the bootc tool applies a container image as an update on an already running Linux system,
+allowing the output to boot an entire OS with the container image included, to test it you either need a dedicated machine or create a
+a virtual machine from the artifact.
```

Comment:
```suggestion
As the bootc tool applies a container image as an update to an already running Linux system, allowing the output to boot an entire OS with the container image included. To test it, you either need a dedicated machine or create a virtual machine from the artifact.
```

Can we be a bit specific about what it refers in the previous statement here?? Do we mean to test the bootc tool?